backend/src/main/java/de/tobias/intestinalinspector/service/AppUserService.java

```diff
@@ -31,4 +32,15 @@ public AppUserEntity create(AppUserEntity newUser) {
     public Optional<AppUserEntity> findUser(String username){
         return appUserRepository.findByUserName(username);
     }
+
+    public AppUserEntity updatePassword(String userName, String setNewPassword) {
```

Comment:
```suggestion
    public AppUserEntity updatePassword(String userName, String newPassword) {
```

setNewPassword sounds like a method